scripts/filter_fdb_entries.py

```diff
@@ -8,33 +8,67 @@
 import traceback
 import time
 
+from ipaddress import ip_address, ip_network, ip_interface
 from collections import defaultdict
 
-def get_arp_entries_map(filename):
+def get_vlan_cidr_map(filename):
+    """
+        Generate Vlan CIDR information from Config DB file
+
+        fdb entries could be contaminated with foreigh Vlan entries as seen in the case of 
+        FTOS fast conversion. SONiC Vlan CIDR configuration will be used to filter out
+        those invalid Vlan entries out.
+
+        Args:
+            filename(str): Config DB data file
+
+        Returns:
+            vlan_cidr(dict) map of Vlan CIDR configuration for SONiC device
+    """
+    with open(filename, 'r') as fp:
+        config_db_entries = json.load(fp)
+
+    vlan_cidr = defaultdict()
+    if "VLAN_INTERFACE" in config_db_entries.keys() and "VLAN" in config_db_entries.keys():
+        for vlan_key in config_db_entries["VLAN_INTERFACE"].keys():
+            vlan, cidr = tuple(vlan_key.split('|'))
```

Comment:
Late comment - There could be entry without cidr in config_db. You may want to skip such entries or it may fail. Can you also put those entries in test_vector and verify. 

            "VLAN_INTERFACE": {
                "Vlan1000": {}
            },